frontend/src/app/main/ui/dashboard/sidebar.cljs

```diff
@@ -513,22 +513,30 @@
        [:li {:on-click (partial on-click :settings-profile)
              :data-test "profile-profile-opt"}
         [:span.icon i/user]
-        [:span.text (tr "labels.profile")]]
-       [:li {:on-click (partial on-click :settings-password)
-              :data-test "password-profile-opt"}
-        [:span.icon i/lock]
-        [:span.text (tr "labels.password")]]
-       [:li {:on-click #(on-click (du/logout) %)
-              :data-test "logout-profile-opt"}
-        [:span.icon i/exit]
-        [:span.text (tr "labels.logout")]]
+        [:span.text (tr "labels.your-account")]]
+       [:li.separator {:on-click #(dom/open-new-window "https://help.penpot.app")
```

Comment:
I think we can leave this address hardcoded. Even for on premise instances, they should use the general Penpot help center and landing pages.

Reply:
:+1: